game/src/sandbox/speed.rs

```diff
@@ -368,6 +371,15 @@ impl JumpToTime {
                         .build(ctx, "close", hotkey(Key::Escape))
                         .align_right(),
                 ]),
+                Checkbox::checkbox(
```

Comment:
Putting this checkbox first is slightly unintuitive to me; it feels like a niche option that should be at the bottom. But I get that having a clear distinction between the two styles of jumping is important, and the checkbox applies to both. What about a column layout:

| jump to time   |  jump to next delay |
-------------------------------------------------
skip drawing

I dunno if that'd look reasonable with the optional agent count picture. Anyway, I think it looks OK for now, just maybe a followup we should try out.

Reply:
Yeah, you're right to call it out, I had similar concerns.

I was surprised to measure that the speed-up is pretty marginal on my machine. I guess drawing isn't very expensive relative to the rest of the sim. We could just get rid of it? 😈

Reply:
I think we added it for #125. On larger maps and slower machines, it makes a difference. I think it's worth keeping, I just want to de-emphasize it, and I think moving it to the bottom would help